src/main/resources/resources.properties

```diff
@@ -6,30 +6,30 @@ test.compound=blaaaah! {0} {1}
+notification.ExerciseNotSelectedNotification.title=No assignment is selected
+notification.ExerciseNotSelectedNotification.content=To submit an assignment, please select the assignment in the list before clicking the submit button.
+notification.FailedToLoadPluginsNotification.title=Error in the A+ Courses plugin
+notification.FailedToLoadPluginsNotification.content=A+ Courses failed to download a list of available IntelliJ plugins from JetBrains'' repository. Please ensure your internet connection is available and restart the IDE. If the problem persists, please contact the course staff.
+notification.MissingFileNotification.title=File not found
+notification.MissingFileNotification.content=The A+ Courses plugin could not find the file {0} in the directory {1}. Please double-check which module you intend to submit and that the appropriate file(s) exist there.
 notification.MissingModuleNotification.title=Could not find module
-notification.MissingModuleNotification.content=A+ Courses plugin couldn''t find the module {0}.
-notification.NetworkErrorNotification.title=A+ Courses plugin encountered a network error
+notification.MissingModuleNotification.content=The A+ Courses plugin could not find the module {0}.
+notification.NetworkErrorNotification.title=The A+ Courses plugin encountered a network error
 notification.NetworkErrorNotification.content=Please check your network connection and try again. Error message: {0}.
-notification.NewModulesVersionsNotification.title=Updates available for A+ Course Modules
-notification.NewModulesVersionsNotification.content=There are newer version(s) for the following A+ Course Modules: {0}.
-notification.NotSubmittableNotification.title=Cannot submit exercise
-notification.NotSubmittableNotification.content=This exercise can only be submitted from the A+ web interface.
+notification.NewModulesVersionsNotification.title=Updates available for course module(s)
+notification.NewModulesVersionsNotification.content=There are newer version(s) of the following course module(s) available: {0}.
+notification.NotSubmittableNotification.title=Assignment not submittable from IDE 
+notification.NotSubmittableNotification.content=This assignment can only be submitted via a web browser on the course web site in A+.
 notification.SubmissionRenderingErrorNotification.title=Failed to open submission
-notification.SubmissionRenderingErrorNotification.content=Failed to open submission. Please open the submission in the A+ web interface. Error message: {0}.
-notification.SuccessfulSubmissionNotification.title=Exercise submitted successfully
-notification.SuccessfulSubmissionNotification.content=You will be notified when feedback for the exercise is available.
+notification.SubmissionRenderingErrorNotification.content=Failed to open the submission in IntelliJ. To check the submission, please use the course web site in A+. Error message: {0}.
```

Comment:
At the moment our "submission rendering" is just opening the submission in the browser, so this text may be a bit misleading at the moment, but it would of course be relevant in the future when we actually render the feedback in IntelliJ.

Reply:
Could change to "Failed to access the submission via IntelliJ"